app/src/main/java/com/kylecorry/trail_sense/tools/battery/ui/FragmentToolBattery.kt

```diff
@@ -11,6 +11,7 @@ import com.kylecorry.andromeda.battery.Battery
 import com.kylecorry.andromeda.battery.BatteryChargingMethod
```

Comment:
In this file, can you add a dialog when the view is created and the battery log is disabled that says the following:

Title: Battery Log
Content: The battery log is disabled, history and time until empty may be unavailable. You can re-enable this in settings.
OK button: Settings (links to the power settings)
Cancel button: Cancel

Reply:
@kylecorry31 now updated. also I used **Settings** for the action instead of **Ok**